src/main/java/io/paradoxical/dropwizard/swagger/SwaggerFilters.java

```diff
@@ -0,0 +1,57 @@
+package io.paradoxical.dropwizard.swagger;
+
+import lombok.Getter;
+import lombok.NonNull;
+
+import java.lang.annotation.Annotation;
+import java.util.function.Predicate;
+
+public class SwaggerFilters {
+
+    @Getter
+    private final Predicate<Class<?>> filterPredicate;
+
+    protected SwaggerFilters(final Predicate<Class<?>> filterPredicate) {
+        this.filterPredicate = filterPredicate;
+    }
+
+    public boolean shouldInclude(Class<?> type) {
+        return filterPredicate.test(type);
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static SwaggerFilters withAnnotation(@NonNull Class<? extends Annotation> annotation) {
+        return builder()
+            .withAnnotation(annotation)
+            .build();
+    }
+
+    public static SwaggerFilters withoutAnnotation(@NonNull Class<? extends Annotation> annotation) {
+        return builder()
+            .withoutAnnotation(annotation)
```

Comment:
this is a side-effect of me changing my intellij setting to get array indentation to be better than the:
```
int[] ints = {
                1,
                2
             }
```